### PR TITLE
enhancement/segmented

### PR DESCRIPTION
**Pull request by nekosaur**
changed VSelect segmented to follow MD spec (#955)

### CODE REVIEW

File: src/components/VSelect/VSelect.vue [hunk 1]
```diff
@@ -138,7 +140,7 @@
         return !this.auto ? items.slice(0, this.lastItem) : items
       },
       isAutocomplete () {
-        return this.autocomplete || this.editable
```
Comment (johnleider) [inline src/components/VSelect/VSelect.vue]: Is there a reason this was removed?

Comment (nekosaur) [reply src/components/VSelect/VSelect.vue]: Yes. It enabled autocomplete when just using editable prop

**Comment by johnleider** [comment]
Editable is autocomplete for all intents and purposes though.

**Comment by johnleider** [comment]
Can you fix conflict and I'll merge

**Pull request merged**